### ISSUE TITLE
After install Trollstore i can’t use the normal appstore or sideload ipas

### ISSUE THREAD

**Issue by EstoyFlaite** [opa334/TrollStore#277]
### No Duplicates Issue

- [X] I'm sure I've searched on the issue tracker before creating one.

### Expected Behavior?

After installing Trollstore everything was going fine, but then I installed an app with the same ID as an AppStore app (WhatsApp Watusi), then I deleted the app via Trollstore.  But now when I try to download WhatsApp from the app store, I can't.  I then checked if WhatsApp was the problem, but no.  I can't install anything, I can't update apps in the AppStore or sideload apps with altstore/sideloadly

 I tried to fix it by restarting the phone, respring and refreshing the app registrations but nothing worked.  I hope you can help me.  Thank you
 
Also, another problem I ran into trying to fix it.  It was that trollstore can't rebuild the icon cache.  At least it shows dark screen and nothing happens (All this with Trollstore 1.1.2 and 1.2.0. I tried both versions) iPhone 13, iOS 15.4.1

### Current Behavior?

Sorry, I put the issue in the other field

### Possible Solution?

-

### Steps to Reproduce

-https://drive.google.com/file/d/1vGp1jJhEJVe6SjVQnAHTl8UbhGXVJUtO/view?usp=drivesdk

### TrollStore Version

TrollInstaller 2 (First release of TrollInstaller 2), Trollstore 1.1.2/Trollstore 1.2.0

### iOS/iPadOS version

15.4.1

### iDevice Model

iPhone 13

**Comment by EstoyFlaite** [comment]
https://drive.google.com/file/d/1vGp1jJhEJVe6SjVQnAHTl8UbhGXVJUtO/view?usp=drivesdk

**Comment by opa334** [comment]
I don't think this issue is related to TrollStore, try signing out of your apple id and signing back in.

**Comment by EstoyFlaite** [comment]
I tried that, changed the connection, checked if I was using a VPN, etc, etc.  I think it's related to Trollstore, my theory is that Trollstore doesn't remove all the files from the app.  Then the AppStore thinks that WhatsApp is installed, but it is not.  And then the entire AppStore has bugs.  Also as I told you, when I try to sideload applications the process fails, whether I use AltStore or the pc, so I'm beginning to doubt that it's just the AppStore.  All this started to happen after installing Trollstore.  If you don't feel like it, that's fine.  But I have a question.

Are there entitlements that can be used by trollstore apps to modify the installation process globally??  Whether it is part of the rights that AppSync uses ??  for instance.  Because the other possibility is that.  That an application is causing this error by trying to modify the behavior.  However I used CocoaTop and I didn't see any app running in the background while the installation problems were happening
